mindocr/data/transforms/rec_transforms.py

```diff
@@ -383,7 +383,7 @@ def __call__(self, data):
 
         # TODO: norm before padding
 
-        data['shape_list'] = [h, w, resize_h / h, resize_w / w] # TODO: reformat, currently align to det
+        data['shape_list'] = np.array([h, w, resize_h / h, resize_w / w])   # TODO: reformat, currently align to det
```

Comment:
Should set as np.float32 (otherwise, it will be float64 and cost more). Actually, GeneratorDataaset will convert them to numpy type in the end.